tools/bin/record_obfuscator.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+
+import hashlib
+import json
+import sys
+from typing import Any
+
+
+def _generate_hash(value: Any) -> str:
+    return hashlib.sha256(str(value).encode()).hexdigest()
+
+
+def obfuscate(value: Any) -> Any:
+    if isinstance(value, str):
+        obfuscated_value = "string_" + _generate_hash(value)
+    elif isinstance(value, (int, float)):
```

Comment:
I think we should keep int and float separate. Json schema has a `integer` type. It'd be more correct to generate float when we see a float and an int when we see an int 😄 .